src/mono/wasm/build/WasmApp.targets

```diff
@@ -195,6 +214,25 @@ EMSCRIPTEN_KEEPALIVE void mono_wasm_load_profiler_aot (const char *desc) { mono_
     <Exec Condition="'$(WasmNativeStrip)' == 'true'" Command="bash -c '$(EMSDK_PATH)/upstream/bin/wasm-opt --strip-dwarf dotnet.wasm -o dotnet.wasm'" IgnoreStandardErrorWarningFormat="true" WorkingDirectory="$(WasmBuildDir)" />
   </Target>
 
+  <Target Name="_GenerateDriverGenC" Condition="'$(RunAOTCompilation)' != 'true' and '$(WasmProfilers)' != ''">
+    <PropertyGroup>
+      <EmccFlags>$(EmccFlags) -DDRIVER_GEN=1</EmccFlags>
+      <InitAotProfilerCmd>
+void mono_profiler_init_aot (const char *desc)%3B
+EMSCRIPTEN_KEEPALIVE void mono_wasm_load_profiler_aot (const char *desc) { mono_profiler_init_aot (desc)%3B }
+      </InitAotProfilerCmd>
```

Comment:
@vargaz given `"'$(RunAOTCompilation)' != 'true' and '$(WasmProfilers)' != '`, could `WasmProfilers` have more than just `aot`? If so, then I should be generating these 2 lines for all of them.

Reply:
Yes, but right now we probably only use aot.